lib/constants/typography.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:flutter/material.dart';
+import 'package:nike_sneakers/constants/app_color.dart';
+
+class AppTypography {
+  static const _raleway = 'Raleway';
+  static const _popoins = 'Popins';
+
+  static const TextStyle ralewayheadingLarge = TextStyle(
+      fontSize: 32,
+      fontWeight: FontWeight.bold,
+      color: AppColor.blackColor,
+      letterSpacing: 1.2,
+      fontFamily: _raleway,
+      height: 1.5);
+  static const TextStyle ralewayheadingSemiLarge = TextStyle(
+      fontSize: 24,
```

Comment:
@CodeWithSuleman 
We have:
Railway 34 bold
Railway 32 bold
Railway 26 semi-bold
Railway 20 semi-bold
Poppins 16 Regular
Poppins 12 Regular